ts/sagas/startup/checkAcknowledgedFingerprintSaga.ts

```diff
@@ -0,0 +1,82 @@
+import { Effect } from "redux-saga";
+import { call, put, select, take } from "redux-saga/effects";
+import { getType } from "typesafe-actions";
+
+import { navigateToOnboardingFingerprintScreenAction } from "../../store/actions/navigation";
+import { fingerprintAcknowledge } from "../../store/actions/onboarding";
+
+import TouchID, { IsSupportedConfig } from "react-native-touch-id";
```

Comment:
all external imports should be at the top

Reply:
fixed